Sources/Purchasing/Configuration.swift

```diff
@@ -174,6 +177,13 @@ import Foundation
             return self
         }
 
+        /// Set `showStoreKitMessagesAutomatically`. Enabled by default. If you want to disable, make sure
+        /// you're configuring the SDK during the `didFinishLaunchingWithOptions` delegate call.
```

Comment:
Let's not mention `didFinishLaunchingWithOptions` here - [it's not the SwiftUI way to do it](https://www.revenuecat.com/docs/configuring-sdk), so it wouldn't apply to a lot of apps. 

Let's instead keep it more high level here, and say "configuring the SDK as early as possible in your app flow, otherwise messages will be displayed automatically". 

Also, I think we could improve this copy significantly - it doesn't really tell you _anything_ about what this is. 

How about: 

```suggestion
        /// Set `showStoreKitMessagesAutomatically`. Enabled by default.
        ///  If enabled, if the user has billing issues, has yet to accept a price increase consent or 
        /// there are other messages from StoreKit, they will be displayed automatically when the app is initialized. 
        /// 
        ///  If you want to disable this behavior so that you can customize when these messages are shown, make sure
        /// you configure the SDK as early as possible in the app's lifetime, 
        /// otherwise messages will be displayed automatically. 
        /// Then use the ``<other method name here>`` method to display the messages. 
        /// More information: <rev.cat link that points [to this page](https://developer.apple.com/documentation/storekit/message)>
```